db/seeds.rb

```diff
@@ -1,4 +1,5 @@
 jcasimir = User.create(username: "jcasimir", password: "test")
+admin = User.create(username: "admin", password: "admin", role: 1)
```

Comment:
If figaro doesn't work, the best bet is to just 'rails c' through heroku and manually make it. In real world applications, there is typically an install process which involves creating an initial superadmin, and they can make more regular admins. Since thats beyond us, I think Figaro or Heroku are our best bet.